ExecCpp/UnitTests/Shu-Osher/prob_parm.H

```diff
@@ -0,0 +1,26 @@
+#ifndef _PROB_PARM_H_
+#define _PROB_PARM_H_
+
+#include <AMReX_REAL.H>
+#include <AMReX_GpuQualifiers.H>
+
+namespace ProbParm {
```

Comment:
This needs to be updated to use the method implemented in https://github.com/AMReX-Combustion/PeleC/pull/234 so we can run on more than just CUDA.

Reply:
ah Jon already flagged that. apologies for dups